port/src/maths.rs

```diff
@@ -0,0 +1,46 @@
+/// Round up by a power of 2
+pub const fn round_up2_usize(n: usize, step: usize) -> usize {
```

Comment:
Is this basically https://doc.rust-lang.org/std/primitive.usize.html#method.next_multiple_of ?

Reply:
hmm, looks like it is.  surprised there's no `previous_multiple_of`